Extension/package.json

```diff
@@ -2761,6 +2761,17 @@
           "default": false,
           "markdownDescription": "%c_cpp.configuration.legacyCompilerArgsBehavior.markdownDescription%",
           "scope": "resource"
+        },
+        "C_Cpp.sshTargetsView": {
+          "type": "string",
+          "enum": [
+            "enabled",
+            "disabled",
+            "auto"
+          ],
+          "default": "auto",
+          "description": "%c_cpp.configuration.sshTargetsView.description%",
+          "scope": "resource"
```

Comment:
Is this the right scope? i.e. does it make sense for 2 different workspace folders to have different values? Maybe it should be "application" or "window" scope? It would be window scope only if it makes sense to change it per workspace.

Reply:
Right, because user probably want to use this feature in one folder/workspace but not in another.

Reply:
But "resource" scope is for when you want 2 different folders in a multi-root workspace to have different values be used, which is unusual for a UI setting, normally it would be "window" scope.